Yoast/wordpress-seo
# Remove functionality to remove blocking XML sitemap files.

## Summary

This PR can be summarized in the following changelog entry:

* Removed the functionality to automatically remove blocking XML sitemap files.

The reasoning is simple: we used to encounter this a lot, we no longer. The code is... non-trivial and _could_ create issues, so better to get rid of it.

## Test instructions

This PR can be tested by following these steps:

* Add a blocking XML sitemap file, see that we no longer do anything. Or check the code and see I only deleted stuff.

## Quality assurance


Fixes #8809
